domain/src/Aggregates/SharePoolingAsset/Entities/SharePoolingAssetTransactions.php

```diff
@@ -24,7 +24,11 @@ public static function make(SharePoolingAssetTransaction ...$transactions): self
         return new self(array_values($transactions));
     }
 
-    /** @return Traversable<int, SharePoolingAssetTransaction> */
+    /**
+     * @return Traversable<int, SharePoolingAssetTransaction>
+     *
+     * @codeCoverageIgnore
+     */
     public function getIterator(): Traversable
```

Comment:
ℹ️ Needs to be implemented because of the `IteratorAggregate` interface, but I don't actually use it anywhere